dcap/types/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rust-version = "1.62.1"
 
 [features]
 default = []
-alloc = ["dep:x509-cert"]
+alloc = ["dep:x509-cert", "serde/alloc"]
```

Comment:
Looks like I missed this in #393.
not fond that `alloc` isn't tested alone. I need to see what cargo release does to find this, as cargo package doesn't seem to notice the lack of dependency when building.

If this continues to be a problem it may be worth looking into something like https://github.com/frewsxcv/cargo-all-features